x-pack/plugins/security_solution/cypress/e2e/data_sources/create_runtime_field.cy.ts

```diff
@@ -16,11 +16,13 @@ import { createCustomRuleEnabled } from '../../tasks/api_calls/rules';
 import { getNewRule } from '../../objects/rule';
 import { refreshPage } from '../../tasks/security_header';
 import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
-import { openEventsViewerFieldsBrowser } from '../../tasks/hosts/events';
 import { assertFieldDisplayed, createField } from '../../tasks/create_runtime_field';
+import { openAlertsFieldBrowser } from '../../tasks/alerts';
+import { deleteAlertsIndex } from '../../tasks/sourcerer';
 
 describe('Create DataView runtime field', () => {
   before(() => {
+    deleteAlertsIndex();
```

Comment:
Why is this used instead of `cleanKibana()` ?

Reply:
Actually, in this particular tests, once a runtime fields is created, you cannot run that test again without starting the cypress instance again because that `runtime` field always exists. I added this so that we can run that test again.

Reply:
@MadameSheema , have improved it now. Please check if it makes sense.